modules/create-policy-categories.adoc

```diff
@@ -1,18 +1,42 @@
 // Module included in the following assemblies:
 //
 // * operating/manage-security-policies.adoc
-:_module-type: PROCEDURE
+:_content-type: PROCEDURE
 [id="create-policy-categories_{context}"]
 = Creating policy categories
 
 You can create new policy categories from the system policies view.
 
+{product-title-short} version 3.74 provides a new method to create and manage policy categories in {product-title-managed} or in {product-title-short} if you have the PostgreSQL database (Technical Preview) enabled. All policy workflows other than policy creation remain unchanged when using this feature. For instructions on using this feature, see the second procedure in the following section.
```

Comment:
"see the second procedure in the following section" is a bit confusing. If possible, I think it would be better to use a reference to the name of the procedure (more on this in the steps below).